modules/network-observability-ebpf-agent-alert.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard-ebpf-agent-alerts_{context}"]
+= Using the eBPF agent alert
+
+When the Network Observability eBPF agent hashmap table is full, the eBPF agent processes flows at a degraded performance. When this is the case, an alert shows `NetObservAgentFlowsDropped`. If you see this alert, consider increasing the `cacheMaxFlows` in the `FlowCollector`, as shown in the following example.
+
+.Procedure
+
+. In the web console, navigate to *Operators* → *Installed Operators*.
+
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+
+. Select *cluster* then select the *YAML* tab.
+
+. Increase the `spec.agent.ebpf.cacheMaxFlows` value, as in the following YAML sample:
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF                                
+    ebpf:
+      cacheMaxFlows: 200000 <1>
+----
+<1> Increase the `cacheMaxFlows` value upwards from its value at the time of the `NetObservAgentFlowsDropped` alert.
```

Comment:
do we want to add a note that this may cause increase in ebpf-agent's memory usage?

Reply:
Good pt. Note added: https://github.com/openshift/openshift-docs/pull/75319/files#diff-905708099880d42a2b5717f69dc04036e708ebec1f9dee4cdb7d58db59614184R10